ironfist/src/cpp/game/save.cpp

```diff
@@ -228,28 +228,63 @@ ironfist_map::hero_t WriteHeroXML(hero* hro) {
 	return hx;
 }
 
+std::string MapVarTypeToString(MapVarType type) {
+	if (type == MAPVAR_TYPE_STRING) {
+		return "string";
+	} else if (type == MAPVAR_TYPE_NUMBER) {
+		return "number";

```

Comment:
There is no integer type in Lua. I guess we could compare a number to its floor, but what would be the necessity behind it?